package.json

```diff
@@ -34,6 +36,7 @@
     "@tippyjs/react": "^4.2.6",
     "@urql/exchange-refocus": "^1.0.0",
     "@walletconnect/web3-provider": "^1.8.0",
+    "apollo-server-core": "^3.12.0",
```

Comment:
Why does this have a different major version from @appollo/server? Did you check these are compatible?

Also did you check with some of the latest commits in the FMDM repo. DeltaDao also did some dep updates, which might be about the same packages if memory serves me correctly

Reply:
DeltaDAO is using the latest version of `apollo`, which is a command line tool that generates the `.d.ts` files from the GraphQL schema. They are using fetch to retrieve the data from the Subgraph. 

I added `apollo-server-core` and `@apollo/server`. Both are the latest version available. The former only adds a plugin to the server: `ApolloServerPluginLandingPageGraphQLPlayground`, which seems to be working fine.

`@apollo/server` replaces `apollo-server-core` so there is no need to keep it. Removed!